Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG OPENVDS_IMAGE=openvds
-ARG ONESEISMIC_BASEIMAGE=golang:1.22-alpine
+ARG ONESEISMIC_BASEIMAGE=docker.io/library/golang:1.22-alpine
```

Comment:
Why is this change needed? Does docker pull the image from another registry if we do not specify `docker.io/library`?